blaze-client/src/test/scala/org/http4s/client/blaze/ExternalBlazeHttp1ClientSpec.scala

```diff
@@ -14,16 +14,14 @@ class ExternalBlazeHttp1ClientSpec extends Http4sSpec with NoTimeConversions wit
     def client = defaultClient
 
     "Make simple http requests" in {
-      val resp = client(uri("https://github.com/")).as[String]
-        .run
+      val resp = client(uri("https://github.com/")).as[String].run
```

Comment:
This isn't new, but just noticing.  Unit tests that depend on an external resource end up being painful in a lot of settings.    I wonder about a helper to fire up our own server to test the client.